src/imgui/ImGuiUtils.hh

```diff
@@ -21,6 +21,8 @@
 #include <string_view>
 #include <utility>
 
+using namespace std::literals;
```

Comment:
Detail: about `using namespace ...`. That's acceptable in `.cc` files but less so in `.hh` files.

The reason is that `using namespace` can change the meaning of code (or can cause ambiguity problems). In a `.cc` file you are in control of what if affected by that `using namespace`, but `.hh` files are (transitively) included in many other files (outside of your control).

Specifically for `using namespace std::literals`, I think the chance of conflicts will be small. And I now see that we (probably I) already do this in some other `.hh` files. But we may still want to avoid this. See below for alternatives.

Reply:
Yes, I considered that, but even the Cpp Core Guidelines make an explicit exception for this specific namespace. And so I thought it would be OK.